Formula/fregerepl.rb

```diff
@@ -0,0 +1,29 @@
+class Fregerepl < Formula
```

Comment:
The homepage and repository URL suggest the file should probably be named `frege-repl` instead, meaning the class name would also need to become `FregeRepl`. Any reason why you decided to omit the dash between `frege` and `repl`?